docs/src/main/asciidoc/spring-di.adoc

```diff
@@ -232,13 +322,18 @@ public class GreeterResource {
 }
 ----
 
-== Update the test
+[id='updating-the-integration-test-of-your-application_{context}']
+== Updating the integration test of your application
```

Comment:
I would rather not use the term `integration` test, as `@QuarkusTest` isn't exactly an integration test (although it's certainly closer to it than to a unit test)